Makefile

```diff
@@ -44,9 +46,23 @@ create-bundle:
 
 .PHONY: verify-bundle
 verify-bundle:
-	cp bundle.tar.gz scripts/bundle.tar.gz
+	cp $(BUNDLE_FILE) scripts/$(BUNDLE_FILE)
 	cd scripts && go run verify-bundle.go
-	rm scripts/bundle.tar.gz
+	rm scripts/$(BUNDLE_FILE)
 
 build-opa:
-	go build ./cmd/opa
\ No newline at end of file
+	go build ./cmd/opa
+
+start-registry:
+	docker run --rm -it -d -p ${REGISTRY_PORT}:5000 --name registry registry:2 
+
+stop-registry:
+	docker stop registry
+
+push-bundle: create-bundle
+	@REPO=localhost:${REGISTRY_PORT}/trivy-checks:latest ;\
+	echo "Pushing to repository: $$REPO" ;\
+	docker run --rm -it --net=host -v $$PWD/${BUNDLE_FILE}:/${BUNDLE_FILE} bitnami/oras:latest push \
```

Comment:
Do we need `--net=host`?

Reply:
This will allow Oras to access the local registry.

Reply:
In my testing, I haven't had the need to allow `--net=host`. 

Run registry:
```
docker run -it --rm -p 6000:5000 registry
```

Push image:
```
oras push localhost:6000/trivy-checks:1 --config /dev/null:application/vnd.cncf.openpolicyagent.config.v1+json  bundle.tar.gz:application/vnd.cncf.openpolicyagent.layer.v1.tar+gzip
```

Reply:
maybe it has something to do with macos?

Reply:
in Makefile oras runs in a container